async-nats/src/client.rs

```diff
@@ -43,6 +44,62 @@ impl From<tokio::sync::mpsc::error::SendError<Command>> for PublishError {
     }
 }
 
+pub struct Publish {
```

Comment:
`#[must_use]` here should prevent mistakenly forgetting to add `.await` at the end.

Reply:
Cool! Added!